client/lib/purchases/actions.js

```diff
@@ -60,6 +62,10 @@ export const submitSurvey = ( surveyName, siteId, surveyData ) => ( dispatch ) =
 		.catch( ( err ) => debug( err ) ); // shouldn't get here
 };
 
+export const cancelPurchaseSurveyTaken = ( purchaseId ) => ( dispatch ) => {
```

Comment:
Small nit, would `cancelPurchaseSurveyCompleted` be better here? I initially read this as `Token` instead of `Taken`.

Reply:
Good point, addressed on https://github.com/Automattic/wp-calypso/pull/94901/commits/ffbcae30eba22d1ac476cb8fa155758e7cf15d7b